libfwupdplugin/fu-self-test.c

```diff
@@ -3401,6 +3401,224 @@ fu_ifd_image_xml_func(void)
 	g_assert_cmpstr(csum1, ==, csum2);
 }
 
+typedef struct {
+	guint last_percentage;
+	guint updates;
```

Comment:
both of these variables could just be 8 bits wide

Reply:
I think keep last_percentage as a guint as then there's no type conversion (i.e. we can detect if we get wonky values)